packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js

```diff
@@ -8,6 +8,9 @@
 
 const app = require('ampersand-app');
 const electron = require('electron');
+const electronDefaultMenu = require('electron-default-menu');
```

Comment:
Electron’s `Menu`’s kinda funny: it’s [apparently read-only](https://github.com/electron/electron/issues/8928#issuecomment-286664574)? This module helps reconstruct the default menu; our “Logs” menu item is added below.